koku/api/metrics/constants.py

```diff
@@ -14,14 +14,30 @@
 OCP_METRIC_MEM_GB_EFFECTIVE_USAGE_HOUR = "memory_gb_effective_usage_per_hour"
 OCP_METRIC_STORAGE_GB_USAGE_MONTH = "storage_gb_usage_per_month"
 OCP_METRIC_STORAGE_GB_REQUEST_MONTH = "storage_gb_request_per_month"
+OCP_NODE_CORE_HOUR = "node_core_cost_per_hour"
 OCP_NODE_MONTH = "node_cost_per_month"
+OCP_NODE_CORE_MONTH = "node_core_cost_per_month"
 OCP_CLUSTER_MONTH = "cluster_cost_per_month"
 OCP_PVC_MONTH = "pvc_cost_per_month"
 
-MEMORY_DISTRIBUTION = "memory"
-CPU_DISTRIBUTION = "cpu"
+# defines the usage type for each metric
+CPU = "cpu"
+MEM = "memory"
```

Comment:
Why were the names here changed? `MEMORY_DISTRIBUTION` and `CPU_DISTRIBUTION` are *actually* descriptive. If I see either of `CPU_DISTRIBUTION` or `MEMORY_DISTRIBUTION` somewhere, I understand its intent. Making these `CPU` and `MEM` strips away meaning.

Reply:
Honestly I don't mind on this, just changed it as Cody suggested. That said every where this is used for a distribution type instead of metric its clearly defined so its probably ok. For example: `self._distribution == metric_constants.MEM`. I have no problem changing it back if people prefer though.

Reply:
Yeah, sorry Luke, you are getting a bit of contradictory opinions here. A little background into my suggestion to just call it `CPU` & `MEM`. Originally within this file we were only utilizing the `cpu` & `memory` strings to represent the different distribution types. 

However, Luke caught some usage metric variables that needed to be added to this constant file. Now those strings (`cpu` & `memory`) can be used to either represent distribution types or usage metric types. Instead of having separate variables that represent the same string, I suggested we have a less descriptive name where those strings are defined then a more descriptive name in the places they are being utilized within this file.  However, totally up for discussion if you disagree.

Reply:
Ok, that's fine. I did not see the convo/suggestions above.